### ISSUE TITLE
Illegal reflective access operations

### ISSUE THREAD

**Issue by dsubelman** [jqno/equalsverifier#379]
**Describe the bug**
When verifying a class that imports class java.text.NumberFormat it throws an illegal reflective access warning for field java.text.NumberFormat.groupingUsed

**To Reproduce**
Verify a class that imports java.text.NumberFormat.

An example would be org.apache.commons.math3.linear.Array2DRowRealMatrix from Apache Commons Math.

**Code that triggers the behavior**
`EqualsVerifier.simple().forClass(Array2DRowRealMatrix.class).verify();`

**Error message**
```
[ERROR] WARNING: An illegal reflective access operation has occurred
[ERROR] WARNING: Illegal reflective access by nl.jqno.equalsverifier.internal.reflection.FieldModifier (file:/Users/dsubelman/.m2/repository/nl/jqno/equalsverifier/equalsverifier/3.5/equalsverifier-3.5.jar) to field java.text.NumberFormat.groupingUsed
[ERROR] WARNING: Please consider reporting this to the maintainers of nl.jqno.equalsverifier.internal.reflection.FieldModifier
[ERROR] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[ERROR] WARNING: All illegal access operations will be denied in a future release
```

**Expected behavior**
no warning

**Version**
3.5

**Additional context**
I haven't test this issue with other classes that import class java.text.NumberFormat.

**Comment by jqno** [comment]
Thanks for letting me know. I've committed a fix. I'll try to make a release soon.

**Comment by jqno** [comment]
It's released: see version 3.5.1.

**Comment by dsubelman** [comment]
Thanks for the super-rapid response.
